packages/lib-auth/src/models/oauth/UIStore.spec.js

```diff
@@ -0,0 +1,98 @@
+import UIStore from './UIStore'
+
+let uiStore
+
+
+describe('Model > UIStore', function () {
+  beforeEach(function () {
+    uiStore = UIStore.create()
+  })
+
+  it('should exist', function () {
+    expect(uiStore).to.not.equal(undefined)
```

Comment:
Same comment about using `expect(object).to.exist` in these tests and if possible, test what it should equal, rather than test that it is not undefined.